### DISCUSSION TITLE
what is the standard about how ArchR automatically annotated peaks

### DISCUSSION THREAD

**Discussion by l-magnificence** [GreenleafLab/ArchR#1869] [Questions / Documentation] [answered]
Who know how ArchR defined peaks as promoters and distal type? Thank you!

**Comment by rcorces** [comment, marked as answer]
That code is in `.fastAnnoPeaks()`. It is based on overlap with annotations present in your `geneAnnotation`
https://github.com/GreenleafLab/ArchR/blob/381842cfef57d61e936df55b29f989ff2eb5e99f/R/ReproduciblePeakSet.R#L604-L653
